SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -2302,6 +2335,33 @@ void ActorSystem::UpdateInterestComponent(AActor* Actor)
 	NetDriver->Connection->SendComponentUpdate(EntityId, &Update);
 }
 
+void ActorSystem::UpdateClientInterest(AActor* Actor, const bool bOverwrite)
+{
+	if (Actor->IsA(APlayerController::StaticClass()) && GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries)
+	{
+		Worker_CommandRequest CommandRequest{};
+		const bool bRequestValid = NetDriver->InterestFactory->CreateClientInterestDiff(Actor, CommandRequest, bOverwrite);
+
+		if (bRequestValid)
+		{
+			const Worker_EntityId SystemEntityId =
+				Cast<USpatialNetConnection>(Actor->GetNetConnection())->ConnectionClientWorkerSystemEntityId;
+
+			NetDriver->Connection->SendCommandRequest(SystemEntityId, &CommandRequest, RETRY_MAX_TIMES, {});
+
+			UE_LOG(LogActorSystem, Log, TEXT("Interest diff: worker entity id %lld"), SystemEntityId);
+		}
+		else
+		{
+			UE_LOG(LogActorSystem, Warning, TEXT("Interest diff: refresh request but no diff detected (%s)"), *Actor->GetName());
+		}
+	}
+	else
+	{
+		UE_LOG(LogActorSystem, Warning, TEXT("Called UpdateClientInterst on non-PlayerController (%s)"), *Actor->GetName());
```

Comment:
NIT Little incorrect since there's also the case the boolean setting isn't enabled

Reply:
This function shouldn't be called unless `bUseClientEntityInterestQueries` is true - will switch to an ensure